modules/skybrowser/include/targetbrowserpair.h

```diff
@@ -0,0 +1,136 @@
+
+    TargetBrowserPair(ScreenSpaceSkyBrowser* browser, ScreenSpaceSkyTarget* target);
+    TargetBrowserPair& operator=(TargetBrowserPair other);
+
+    // Target & Browser
+    void initialize();
+    // Highlighting
+    void removeHighlight(glm::ivec3 color); 
+    void highlight(glm::ivec3 color);
+    // Animation
+    void startAnimation(glm::dvec3 coordsEnd, float fovEnd, bool shouldLockAfter = true);
+    void incrementallyAnimateToCoordinate(double deltaTime);
+    void incrementallyFade(float goalState, float fadeTime, float deltaTime);
+    // Mouse interaction
+    bool checkMouseIntersection(glm::vec2 mousePosition);
+    glm::vec2 selectedScreenSpacePosition();
+    bool isBrowserSelected();
+    bool isTargetSelected();
+    void fineTuneTarget(const glm::vec2& start, const glm::vec2& translation);
+    void translateSelected(const glm::vec2& start, const glm::vec2& translation);
+    void synchronizeAim();
+
+    // Browser
+    void sendIdToBrowser();
+    void updateBrowserSize();
+
+    // Target
+    void centerTargetOnScreen();
+    void lock();
+    void unlock();
+
+    bool hasFinishedFading(float goalState) const;
+    bool isFacingCamera() const;
+    bool isUsingRadiusAzimuthElevation() const;
+    bool isEnabled() const;
+    bool isLocked() const;
+
+    void setEnabled(bool enable);
+    void setIsSyncedWithWwt(bool isSynced);
+    void setVerticalFov(float vfov);
+    void setEquatorialAim(const glm::dvec2& aim);
+    void setBorderColor(const glm::ivec3& color);
+    void setScreenSpaceSize(const glm::vec2& dimensions);
+    void setVerticalFovWithScroll(float scroll);
+    void setSelectedWithId(const std::string& id);
+
+    float verticalFov() const;
+    glm::ivec3 borderColor() const;
+    glm::dvec2 targetDirectionEquatorial() const;
+    glm::dvec3 targetDirectionGalactic() const;
+    std::string browserGuiName() const;
+    std::string browserId() const;
+    std::string targetId() const;
+    std::string selectedId();
+    glm::vec2 size() const;
+    
+    ScreenSpaceSkyTarget* getTarget();
+    ScreenSpaceSkyBrowser* getBrowser();
+    const std::deque<int>& getSelectedImages() const;
+    
+    // WorldWide Telescope image handling
+    void setImageOrder(int i, int order);
+    void selectImage(const ImageData& image, int i);
+    void removeSelectedImage(int i);
+    void loadImageCollection(const std::string& collection);
+    void setImageOpacity(int i, float opacity);
+    void hideChromeInterface(bool shouldHide);
+
+    friend bool operator==(const TargetBrowserPair& lhs, 
+                           const TargetBrowserPair& rhs);
+    friend bool operator!=(const TargetBrowserPair& lhs, 
+                           const TargetBrowserPair& rhs);
+
+private:
+    bool isTargetFadeFinished(float goalState) const;
+    bool isBrowserFadeFinished(float goalState) const;
+
+    // Selection
+    ScreenSpaceRenderable* _selected{ nullptr };
```

Comment:
We usually don't use braces for initialization. Use `=` instead:
```
ScreenSpaceRenderable* _selected = nullptr;
```

The same goes for all the other member variables in this file